awx/conf/views.py

```diff
@@ -120,7 +121,11 @@ def perform_update(self, serializer):
                 setting.save(update_fields=['value'])
                 settings_change_list.append(key)
         if settings_change_list:
-            connection.on_commit(lambda: handle_setting_changes.delay(settings_change_list))
+            connection.on_commit(lambda: clear_setting_cache.delay(settings_change_list))
+            if any([setting.startswith('LOG_AGGREGATOR') for setting in settings_change_list]):
```

Comment:
I know this was just copied and modified from the original code in system.py, but is there any reason to build up a new list here and not use `any` with a generator? Like `any(setting.startswith('LOG_AGGREGATOR') for setting in settings_change_list)`

I guess the list of changed settings will be tiny enough in all cases that it doesn't matter much, though.

Reply:
I don't think so. ```any``` with a generator gets us to the same result. It's only a list of 14 settings when I test and I don't think the number of settings would reach a size where it would have much impact.